Source/WebCore/page/Performance.cpp

```diff
@@ -281,6 +289,29 @@ void Performance::countEvent(EventType type)
     eventCounts()->add(type);
 }
 
+void Performance::processEventEntry(PerformanceEventTiming::Candidate& candidate, DOMHighResTimeStamp duration)
+{
+    // FIXME: handle firstInput based on InteractionId
+    if (!m_firstInput) {
+        m_firstInput = PerformanceEventTiming::create(candidate, duration, true);
+        queueEntry(*m_firstInput);
+    }
+
+    if (duration < 16.0)
+        return;
+
+    // We could do an even better of returning early to avoid the creation of
```

Comment:
even better job? I don't think this comment is necessary.
If anything, we should add FIXME instead:
```
// FIXME: Implement a more precise early return.
```